releasenotes/notes/downgrade-encoding-to-v05-96458e1942295721.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    tracing: Encoding traces in the v05 format has known issues for applications generating spans at a frequency.
```

Comment:
can we be more specific than "a frequency" ?

Reply:
We can reword this to say 1/1000000 spans